client/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,104 @@
+input,
+textarea {
```

Comment:
When styling tags directly, I'd create a partial under ```base```. When styling classes, then I'd add it as a component.